pls/backend.go

```diff
@@ -159,6 +159,10 @@ func New(ctx *node.ServiceContext, config *Config) (*Plasma, error) {
 		}
 		cacheConfig = &core.CacheConfig{Disabled: config.NoPruning, TrieCleanLimit: config.TrieCleanCache, TrieDirtyLimit: config.TrieDirtyCache, TrieTimeLimit: config.TrieTimeout}
 	)
+
+	pls.chainConfig.RootchainAddress = config.RootChainContract
+	pls.chainConfig.RootchainURL = config.RootChainURL
```

Comment:
`chainConfig.RootchainAddress` and `chainConfig.RootchainURL` is duplicated with `pls.Config`.